examples/app-with-extension-point/tests/extendActions.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@grafana/plugin-e2e';
```

Comment:
Was thinking about wether or not to keep the testIds. What do you think about replacing them with other selectors such as getByRole or getByLabel?

Reply:
There are no form elements where labels are associated with inputs. Could potentially use getByRole in some places, but I don't see any harm in using test ids here.